server/services/git/git_service.py

```diff
@@ -0,0 +1,49 @@
+from flask import current_app
+
+
+from server.services.git.file_service import FileService
```

Comment:
Is It possible to have relative imports in Flask? Something like `from file_service import FileService`.

I like to organize the imports that way:

```
python standard libraries imports

main framework imports

third party libs

project imports
```

like on: https://github.com/willemarcel/osmcha-django/blob/master/osmchadjango/changeset/views.py#L2-L40

Reply:
I tried to change to relative imports, but it doesn't work.

I will update the import order